packages/macos/package_files/build.sh

```diff
@@ -11,6 +11,7 @@ set -exf
 DESTINATION_PATH=$1
 SOURCES_PATH=$2
 BUILD_JOBS=$3
+DEBUG_ENABLED=$4
```

Comment:
Could we just use DEBUG? It seems like it's always the same flag, and the current name is a bit confusing.